lib/html_scrubber_util.rb

```diff
@@ -9,8 +9,15 @@ module HtmlScrubberUtil
   def sanitize_html(value)
     ActionController::Base.helpers.sanitize(
       value,
-      tags: Loofah::HTML5::SafeList::ACCEPTABLE_ELEMENTS.dup.delete("select").add("style"),
-      attributes: Loofah::HTML5::SafeList::ACCEPTABLE_ATTRIBUTES.dup.add("style")
+      tags: Loofah::HTML5::SafeList::ACCEPTABLE_ELEMENTS.dup.delete('select').add('style'),
+      attributes: Loofah::HTML5::SafeList::ACCEPTABLE_ATTRIBUTES.dup.merge(
+        [
+          'style',
+          'data-toggle',
+          'data-slide-to',
+          'data-target'
```

Comment:
Could we add specs for these in https://github.com/ideacrew/enroll/blob/trunk/spec/lib/html_scrubber_util_spec.rb?

Reply:
Added!